spec/integration/integration_spec_helper.rb

```diff
@@ -64,6 +65,48 @@ def normalize_inspect_output(output)
     gsub!(/(\r\033\[K.*?\r\033\[K).*\r\033\[K(.*)/, "\\1\\2") # strip all progress messages but two
 end
 
+def machinery_boxes
+  {
+    "openSUSE_13_2" => "machinery_132",
+    "openSUSE_13_1" => "machinery_131",
+    "Tumbleweed" => "machinery_tumbleweed"
+  }
+end
+
+def boxes
+  {
+    "openSUSE_13_2" => "opensuse132",
+    "openSUSE_13_1" => "opensuse131"
+  }
+end
+
+def test_according_to_matrix(currently_running_on)
+  matrix_path = File.join(Machinery::ROOT, "..", "alfred", "documents", "source", "integration_tests.yml")
+  matrix = YAML.load_file(matrix_path)
+
+  matrix.each do |command|
+    command_name, support_level = command
+    support_level.each do |type, test_relations|
```

Comment:
Unused block argument - `type`. If it's necessary, use `_` or `_type` as an argument name to indicate that it won't be used.